geometry_msgs/Pose grasp_pose
string grasp_part_affordance
int32 score